plugins/dash/dash.plugin.zsh

```diff
@@ -0,0 +1,86 @@
+# Usage: dash [keyword:]query
+dash() { open dash://"$*" }
+compdef _dash dash
+
+_dash() {
+  # No sense doing this for anything except the 2nd position and if we haven't
+  # specified which docset to query against
+  if [[ $CURRENT -eq 2 && ! "$words[2]" =~ ":" ]]; then
+    local -a _all_docsets
+    _all_docsets=()
+    # Use defaults to get the array of docsets from preferences
+    # Have to smash it into one big line so that each docset is an element of
+    # our DOCSETS array
+    DOCSETS=("${(@f)$(defaults read com.kapeli.dash docsets | tr -d '\n' | grep -oE '\{.*?\}')}")
```

Comment:
From my command line this is wrong. Should be:

**com.kapeli._dashdoc_**

Typo or did the key change in Dash recently?